### ISSUE TITLE
Support SBT

### ISSUE THREAD

**Issue by adericbourg** [dependencyci/support#3]
SBT is not in the list of supported tools.

Any plan for that on your roadmap?

**Comment by andrew** [comment]
Yeah it's on the roadmap, not there yet as it's a little more tricky to parse dependencies from without executing the file.

**Comment by pedrorijo91** [comment]
any time estimation for this? :)

**Comment by andrew** [comment]
@pedrorijo91 no definitive timeline yet, will update when I know more
